src/feature/flow_chart_panel/hooks/useAddNewNode.tsx

```diff
@@ -60,6 +61,7 @@ export const useAddNewNode = () => {
                 funcName === "CONSTANT"
                   ? nodeLabel
                   : params![param].default?.toString(),
+              valType: "unknown" as ParamValueType,
```

Comment:
Could you not get the correct type from `params`? e.g `params[param].type`

Reply:
RIght! I wrote this as a placeholder and forgot about it. Tysm!